moveit_ros/planning/robot_model_loader/src/robot_model_loader.cpp

```diff
@@ -110,10 +110,21 @@ void RobotModelLoader::configure(const Options& opt)
     for (moveit::core::JointModel* jmodel : model_->getJointModels())
     {
       std::vector<moveit_msgs::JointLimits> jlim = jmodel->getVariableBoundsMsg();
+			std::vector<double> jw = jmodel->getDistanceFactor();			
```

Comment:
I can't stand one/two letter var names. I just did a cleanup PR that should be merged before yours:
https://github.com/ros-planning/moveit/pull/1759

Reply:
Done. Changed all the names to the way you have updated